docker/README.md

```diff
@@ -20,22 +20,22 @@ docker volume create drand
 Next we must create a keypair and store it in the docker volume we've just created.
 
 ```shell
-docker run --rm --volume drand:/data/drand drandorg/go-drand:v1.5.3 generate-keypair  --folder /data/drand/.drand --tls-disable --id default 0.0.0.0:8080
+docker run --rm --volume drand:/data/drand drandorg/go-drand:v1.5.3 generate-keypair  --folder /data/drand/.drand --id default 0.0.0.0:8080
```

Comment:
I have no clue if we want to keep it or remove it there

Reply:
let's leave this one, so LoE members running docker keep TLS on